### ISSUE TITLE
It isn't working anymore

### ISSUE THREAD

**Issue by peter-ch** [takluyver/nbopen#82]
I updated the Jupyter notebook to 7.0.2 and nbopen isn't working at all anymore, reinstalling didn't help.

**Comment by sbushmanov** [comment]
Just to add error message:

```
nbopen FakeGen_CSV.ipynb                                                                                                                                                        (base)
Traceback (most recent call last):
  File "/home/sergey/anaconda3/bin/nbopen", line 5, in <module>
    from nbopen import main
  File "/home/sergey/anaconda3/lib/python3.9/site-packages/nbopen/__init__.py", line 5, in <module>
    from .nbopen import main
  File "/home/sergey/anaconda3/lib/python3.9/site-packages/nbopen/nbopen.py", line 7, in <module>
    from notebook import notebookapp
ImportError: cannot import name 'notebookapp' from 'notebook' (/home/sergey/anaconda3/lib/python3.9/site-packages/notebook/__init__.py)
```

**Comment by takluyver** [comment]
Right, sorry about that. I believe notebook 7 is rebuilt on top of Jupyterlab, so it presumably needs some changes. I'm currently not really using nbopen - I'm using notebooks on a server rather than locally - so I'm unlikely to get to this, but if someone can figure out a reasonably simple change and make a PR, I can hopefully take a look.

**Comment by takluyver** [comment]
Thanks @jfeist for the fix. I've just released 0.7 to PyPI with PR #83 merged.

**Comment by peter-ch** [comment]
Thanks!
